dashboard/lib/views/index.jade

```diff
@@ -3,24 +3,19 @@ extends layout
 block content
 
   block container
-      div.box0
-          .header
-              .header-content
-                p SANITY CHECK STATUS | OVERVIEW
 
-
-          div.box1
+            div#frameContainer(style="height:100%")
+            div.box1#box1
 
              #box2.box2
+                p
                 ul.region_list
                    each region in regions
                       - color = region['status'] == 'OK' ?'green' : region['status'] == 'NOK'?'red':'yellow';
```

Comment:
now we have 4 states
- green  -> ok
- red -> nok
- white or grey -> N/A
- yellow -> pok